alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/AlfrescoStatusMetrics.java

```diff
@@ -0,0 +1,63 @@
+package eu.xenit.alfred.telemetry.binder;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import org.alfresco.repo.security.authentication.AuthenticationUtil;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.cmr.admin.RepoAdminService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+
+public class AlfrescoStatusMetrics implements MeterBinder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AlfrescoStatusMetrics.class);
+    private static final String STATUS_PREFIX = "alfresco.status";
+
+    private RepoAdminService repoAdminService;
+    private RetryingTransactionHelper retryingTransactionHelper;
+
+    public AlfrescoStatusMetrics(RepoAdminService repoAdminService, RetryingTransactionHelper retryingTransactionHelper) {
+        this.repoAdminService = repoAdminService;
+        this.retryingTransactionHelper = retryingTransactionHelper;
+    }
+
+    @Override
+    public void bindTo(@Nonnull MeterRegistry meterRegistry) {
+        LOGGER.info("Registering Alfresco Status metrics");
+        Gauge.builder(String.format("%s.%s", STATUS_PREFIX, "readonly"),
```

Comment:
Why the `String.format(...)`? IMO `STATUS_PREFIX + ".readOnly` is easier to read?

Reply:
Ok, I corrected.